voila/handler.py

```diff
@@ -69,7 +69,7 @@ def get(self, path=None):
         }
 
         # include potential extra resources
-        extra_resources = self.voila_configuration.resources
+        extra_resources = self.voila_configuration.config.VoilaConfiguration.resources.to_dict()
```

Comment:
Which addition? The `.config.VoilaConfiguration` part or the `.to_dict()`?

I'm noticing that it would be smarter to have the latter two lines afterwards, so the `to_dict()` method would be called *only* when entering the `if` block.